tests/service/data/SyntaxTree/IfThenElse/Comment after else 02.fs.bsl

```diff
@@ -0,0 +1,26 @@
+ImplFile
+  (ParsedImplFileInput
+     ("/root/IfThenElse/Comment after else 02.fs", false,
+      QualifiedNameOfFile Comment after else 02, [], [],
+      [SynModuleOrNamespace
+         ([Comment after else 02], false, AnonModule,
+          [Expr
+             (IfThenElse
+                (Ident a,
+                 ArbitraryAfterError ("typedSequentialExprBlock1", (1,9--1,9)),
+                 None, Yes (1,0--1,9), false, (1,0--1,9),
+                 { IfKeyword = (1,0--1,2)
+                   IsElif = false
+                   ThenKeyword = (1,5--1,9)
+                   ElseKeyword = None
+                   IfToThenRange = (1,0--1,9) }), (1,0--1,9));
+           Expr (Ident b, (2,0--2,1))], PreXmlDocEmpty, [], None, (1,0--3,0),
+          { LeadingKeyword = None })], (true, true),
+      { ConditionalDirectives = []
+        CodeComments = [BlockComment (3,5--3,33)] }, set []))
+
+(2,0)-(2,1) parse error Possible incorrect indentation: this token is offside of context started at position (1:1). Try indenting this token further or using standard formatting conventions.
+(2,0)-(2,1) parse error Expecting expression
+(3,0)-(3,36) parse error Unexpected keyword 'elif' in implementation file
```

Comment:
Where did `elif` come from? It doesn't seem to be in the implementation file.

Reply:
`LexFilter` replaces `else` + `if` with `elif`.

Reply:
https://github.com/dotnet/fsharp/blob/e333828359e765d6d39e290141dcde9d08c9f02b/src/Compiler/SyntaxTree/LexFilter.fs#L2294

Reply:
Might be confusing to users if they see it. But I suppose that didn't change with this PR.